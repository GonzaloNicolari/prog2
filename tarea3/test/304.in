# prueba de remover_de_binario
insertar_en_binario (6,r)
remover_de_binario r
imprimir_binario
insertar_en_binario (6,r)
insertar_en_binario (6,y)
remover_de_binario y
imprimir_binario
insertar_en_binario (6,m)
remover_de_binario m
imprimir_binario
insertar_en_binario (6,y)
remover_de_binario r
imprimir_binario
insertar_en_binario (6,m)
remover_de_binario y
imprimir_binario
insertar_en_binario (6,d)
insertar_en_binario (6,y)
remover_de_binario m
imprimir_binario
insertar_en_binario (6,m)
insertar_en_binario (6,r)
insertar_en_binario (6,o)
insertar_en_binario (6,p)
insertar_en_binario (6,g)
insertar_en_binario (6,j)
insertar_en_binario (6,i)
insertar_en_binario (6,h)
insertar_en_binario (6,l)
insertar_en_binario (6,k)
imprimir_binario
remover_de_binario m
imprimir_binario
Fin
