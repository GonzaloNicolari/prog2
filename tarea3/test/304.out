1>#  prueba de remover_de_binario.
2>Insertado.
3>Removido.
4>
5>Insertado.
6>Insertado.
7>Removido.
8>
(6,r)
9>Insertado.
10>Removido.
11>
(6,r)
12>Insertado.
13>Removido.
14>
(6,y)
15>Insertado.
16>Removido.
17>
(6,m)
18>Insertado.
19>Insertado.
20>Removido.
21>
-(6,y)
(6,d)
22>Insertado.
23>Insertado.
24>Insertado.
25>Insertado.
26>Insertado.
27>Insertado.
28>Insertado.
29>Insertado.
30>Insertado.
31>Insertado.
32>
-(6,y)
---(6,r)
-----(6,p)
----(6,o)
--(6,m)
-----(6,l)
------(6,k)
----(6,j)
-----(6,i)
------(6,h)
---(6,g)
(6,d)
33>Removido.
34>
-(6,y)
---(6,r)
-----(6,p)
----(6,o)
--(6,l)
-----(6,k)
----(6,j)
-----(6,i)
------(6,h)
---(6,g)
(6,d)
35>Fin.
